### PR TITLE
BaSP-MR-144: Activities and Super-Admins react hook form implementation

### PR DESCRIPTION
**Pull request by lorenzomauro59**
- Implement shared input 
- Update all current forms and shared components used in the forms (inputs). Take into account styles for errors
- Adding validations to forms (required fields, dates, names, number of characters). Using the JOI library)
- Add reset function button to clear fields

### CODE REVIEW

File: src/Components/Activities/Form/index.jsx [hunk 1]
```diff
@@ -1,56 +1,63 @@
...
+  } = useForm({
+    mode: 'onBlur',
+    resolver: joiResolver(activityValidation)
+  });
+
+  const getActivityData = async () => {
+    try {
+      const response = await dispatch(getActivitiesById(id));
+      const activityData = response.data;
+      console.log(activityData);
```
Comment (ghost) [inline src/Components/Activities/Form/index.jsx]: remember remove this console.log

**Review by marcohzo — APPROVED** [review]

**Review by ghost — APPROVED** [review]
Grat job!!

**Review by ale-schmidt — APPROVED** [review]
Looks and works perfect!

**Review by fernandofernandezuy — APPROVED** [review]
Great job!

**Review by EmaBaldor — APPROVED** [review]
Great Job!

**Comment by lorenzomauro59** [comment]
[[Activities & Super Admins] - React-hook-form implementation](https://trello.com/c/gyk7gv3i/144-activities-super-admins-react-hook-form-implementation)

**Comment by araefedg** [comment]
Everything implemented seems to work correctly, meets all the requirements and passes the test, great work!

**Pull request merged**